### ISSUE TITLE
Support TypeVar

### ISSUE THREAD

**Issue by mtshiba** [mtshiba/pylyzer#5]

**Comment by darkvertex** [comment]
Boo! :( I was excited to use pylyzer but this missing feature is a dealbreaker for me.

Here's sample code to reproduce the issue:

```python
from typing import TypeVar

class Vegetable:
    pass

class Potato(Vegetable):
    pass

class Carrot(Vegetable):
    pass

T = TypeVar('T', Potato, Carrot)

def get_vegetable() -> T:
    # Return an instance of either Potato or Carrot
    return Potato()
```

pylyzer output without any special arguments:
```
Found 3 errors: blabla.py
Error[#0591]: File ./fp_link_backend/sql_app/blabla.py, line 12, <module>.T

12 | T = TypeVar('T', Potato, Carrot)
   :     -------
   :           |- expected: Subroutine
   :           `- but found: {typing.TypeVar}

TypeError: the type of TypeVar is mismatched

Error[#0596]: File ./fp_link_backend/sql_app/blabla.py, line 14, <module>

14 | def get_vegetable() -> T:
   :                        -

NameError: Type T is not defined

Error[#0596]: File ./fp_link_backend/sql_app/blabla.py, line 14, <module>.get_vegetable

14 | def get_vegetable() -> T:
   :                        -
   :                        `- exists a similar name type: T

NameError: Type T is not defined
```

**Comment by mtshiba** [comment]
implemented in v0.0.60 (ce122851437e5cd44fde78f8e27abfaabc4a91d9).
